kustomize/commands/internal/util/configmapSecretFlagsAndArgs.go

```diff
@@ -1,54 +1,57 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-package add
+package util
```

Comment:
suggestion: separate out this change in its own PR, so that this PR focuses on just adding the `edit set configmap` feature

Reply:
Will do!

Reply:
Extracted this to a new PR: #5409. Also undid the renames.

Reply:
Approved. Thanks!

Reply:
Rebased this to pick up the new changes!